### ISSUE TITLE
I'm not seeing any BG data

### ISSUE THREAD

**Issue by NightScout87** [nightscout/cgm-remote-monitor#6319]
Good day all,

I used to use Spike app + Nightscout and it worked like a charm. 

I am now using the Dexcom app. I started at Step 3 from this guide https://nightscout.github.io/nightscout/new_user since I already have an account for GitHub and Heroku.

When I connect to NightScout, I don't see any data. 

From Papertrail I see the following error message in loop:

![image](https://user-images.githubusercontent.com/56931190/97519064-f6a46d00-196e-11eb-8fdf-24a078814650.png)

all buckets are empty

I've followed this troubleshooting guide https://nightscout.github.io/troubleshoot/troublehoot/#no-data-in-nightscout but I nothing worked.

I'm currently using sugarmate as a workaround but I miss NightScout.

Does anyone has any idea what's going on?

**Comment by psonnera** [comment]
@NightScout87 can you confirm you checked no special characters in your Clarity credentials and tried to create another account? https://nightscout.github.io/troubleshoot/troublehoot/#dexcom-username-issue You do have bridge in the enable variable? Can you manually add treatments in Nightscout (just to test not read-only)?

**Comment by NightScout87** [comment]
@psonnera Yes I confirm I do not have any special characters in my Clarity account. I also to create another account like you asked it did not work.

When you say enable variable, are you talking about those value:

![image](https://user-images.githubusercontent.com/56931190/97793305-b89e8780-1bc0-11eb-90b1-5122de7e0e38.png)

I am able to add treatments in Nightscout mannually.


Thanks!
